src/css/adoc/table.css

```diff
@@ -204,9 +204,42 @@ div.panels table {
     border-top: unset;
     display: block;
     font-size: 14px;
+    margin-left: 0;
+    margin-right: 0;
     margin-top: var(--lg);
     overflow: auto;
-    width: 100%;
+    width: fit-content;
+
+    &.grid-none {
+      & > * > tr {
+        border: none !important;
+
+        & > td {
+          border: none;
+        }
+      }
+    }
+
+    &.grid-rows {
+      td {
+          border: 1px;
+        }
+    }
+
+    &.grid-cols {
+      & tr {
+        border-top: unset !important;
+      }
+      
+      & td {
+        border-left: 1px solid #cacbcc;
+        border-right: 1px solid #cacbcc;
+      }
+    }
```

Comment:
This does not work because it causes outer borders.

To remove the outer border, do the following:
- change `td` to `td+td` to specify for all <td>s except for the first (leftmost column)
-  remove border-right property

then for clean up, use `var(--aluminum-4)` instead of #cacbcc.